src/main/java/net/snowflake/client/core/SFSessionProperty.java

```diff
@@ -55,7 +55,8 @@
   INJECT_WAIT_IN_PUT("inject_wait_in_put", false, Integer.class),
   PRIVATE_KEY_FILE("private_key_file", false, String.class),
   PRIVATE_KEY_FILE_PWD("private_key_file_pwd", false, String.class),
-  CLIENT_INFO("snowflakeClientInfo", false, String.class);
+  CLIENT_INFO("snowflakeClientInfo", false, String.class),
+  ALLOW_UNDERSCORES_IN_HOST("allowUnderscoresInHost", false, Boolean.class);
```

Comment:
Are we adding this parameter for customers? If so, are we sure adding the parameter here is enough to expose it to client side?

Reply:
this should be sufficient